docs/meshcloud.customer.md

```diff
@@ -17,12 +17,49 @@ If you would like to give others access to your meshCustomer and the related mes
 
 ## Assign User Roles
 
-You can change the group in the dropdown and save the changes via the *disc* icon. The following groups are available:
+You can change the meshCustomer role for each user in the dropdown and save the changes via the *disc* icon. The following roles are available:
```

Comment:
You can change the role of each user associated with a meshCustomer in the dropdown and save the changes via  *disc* icon. The following roles are available: